corehq/apps/cleanup/management/commands/delete_es_docs_in_domain.py

```diff
@@ -0,0 +1,27 @@
+import sys
+
+from django.core.management import BaseCommand
+
+from corehq.apps.cleanup.tasks import _get_missing_domains
+from corehq.apps.es import AppES, CaseES, CaseSearchES, FormES, GroupES, UserES
+from corehq.apps.es.registry import registry_entry
+from corehq.apps.es.transient_util import doc_adapter_from_info
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+
+    def handle(self, domain, **options):
+        deleted_domains = _get_missing_domains()
```

Comment:
It might be nicer to just look up the domain they put in and see if it is deleted

Reply:
Ha good point. See b119b12e25af966589f103c304cce583d6a4be3a